examples/warehouse/_config/README

```diff
@@ -0,0 +1,14 @@
+Tool Configuration
+================
+
+Type name: _config
+
+Represents warehouse specific configuration for a palletjack tool.
+
+Contains the tool specific information needed to run a palletjack tool
+against a certain warehouse.
+
+
+Files:
+
+  _config/<name>/<somecfg>.yaml
```

Comment:
This should be clarified to note that the relevant name is that of the tool class.